draft-ietf-quic-transport.md

```diff
@@ -1474,31 +1506,33 @@ validate a client address, it provides the client with a token.  As long as the
 token cannot be easily guessed (see {{token-integrity}}), if the client is able
```

Comment:
s/as the token cannot be easily guessed/as the token's authenticity can be checked/